docs/user-guide/installation.rst

```diff
@@ -61,12 +204,11 @@ will help you pick which pipeline to use.
     If you want to make sure you have the dependencies
     installed for any component you might ever need, and you
     don't mind the additional dependencies lying around, you can use
+    this to install everything:
```

Comment:
should probably mention here they they need to clone the repo to get this requirements file.